.env.example

```diff
@@ -9,7 +9,7 @@ ENABLE_API_DOCS=True
 DOCUMENT_CACHE_TTL_MS=0
 
 # Vespa config
-VESPA_URL=http://vespatest:19071
+VESPA_URL=http://vespatest:8080
```

Comment:
Is it a good idea to maybe keep the other URL as a separate variable and a comment for each to differentiate what the URLs are for - not sure if that URL was needed for dev work by the platform team at some point?